org-ql-search.el

```diff
@@ -271,23 +280,30 @@ Valid parameters include:
 For example, an org-ql dynamic block header could look like:
 
   #+BEGIN: org-ql :query (todo \"UNDERWAY\") :columns (priority todo heading) :sort (priority date) :ts-format \"%Y-%m-%d %H:%M\""
-  (-let* (((&plist :query :columns :sort :ts-format :take) params)
+  (-let* (((&plist :scope :query :columns :sort :ts-format :take) params)
           (query (cl-etypecase query
-                   (string (org-ql--query-string-to-sexp query))
+                   (string (org-ql--plain-query query))
```

Comment:
This change probably indicates that this patch is based on an outdated version of org-ql.